.azure/autotest_template.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  action: 'fly'
+
+jobs:
+- job: sitl_${{ parameters.name }}_test
+  displayName: ${{ format('Cygwin SITL {0} test', parameters.name) }}
+  pool:
+    vmImage: 'win1803'
+  condition: eq(variables['Build.Reason'], 'Schedule')
+  steps:
+  - script: choco install cygwin --params "/InstallDir:C:\Cygwin /NoStartMenu /NoAdmin"
+    displayName: 'Install Cygwin'
+  - script: choco install gcc-g++ python2 python2-devel python2-future python2-lxml python2-pip python2-pexpect python2-numpy git gettext --source cygwin
+    displayName: 'Install Cygwin packages'
+  - script: git submodule update --recursive --init modules/mavlink
+    displayName: Initialize MAVLink submodule
+  - script: C:\Cygwin\bin\bash --login -c "cd $(cygpath '%BUILD_SOURCESDIRECTORY%')/modules/mavlink/pymavlink && python setup.py build install --user"
+    displayName: 'Install pymavlink from submodule'
+  - script: C:\Cygwin\bin\bash --login -c "pip2 install MAVProxy"
```

Comment:
I would add -U in case the VM wasn't really clean and we need a newer mavproxy, no ?

Reply:
VM is always clean, it's a new VM each time.